app/assets/stylesheets/views/_answers.scss

```diff
@@ -0,0 +1,9 @@
+.answers {
+  .add-title-margin {
+    @include responsive-top-margin;
+  }
+
+  strong {
```

Comment:
@fofr what does that do, this fails the grep test
https://github.com/search?q=org%3Aalphagov+rich_govspeak&type=Code